src/realm-settings/RealmSettingsSection.tsx

```diff
@@ -164,22 +164,30 @@ export const RealmSettingsSection = () => {
   const [currentUser, setCurrentUser] = useState<UserRepresentation>();
   const { whoAmI } = useWhoAmI();
 
-  const kpComponentTypes =
-    useServerInfo().componentTypes!["org.keycloak.keys.KeyProvider"];
+  const kpComponentTypes = useServerInfo().componentTypes![KeyProviderType];
```

Comment:
It might be better to use optional chaining with a default value here.
```suggestion
  const kpComponentTypes = useServerInfo().componentTypes?.[KeyProviderType] ?? [];
```